events/interaction_events.go

```diff
@@ -42,6 +42,23 @@ func (e *ApplicationCommandInteractionCreate) Guild() (discord.Guild, bool) {
 	return discord.Guild{}, false
 }
 
+// Acknowledge acknowledges the interaction.
+//
+// This is used strictly for acknowledging the HTTP interaction request from discord. This responds with 202 Accepted.
+//
+// When using this, your first http request must be <event>.Client().Rest().CreateInteractionResponse()
```

Comment:
you can create a go doc link via `[rest.Interactions.CreateInteractionResponse]` afaik
this should also include `[rest.Interactions.CreateInteractionResponseWithCallback]`

Reply:
I am pretty sure you don't even need to link to pkg.go.dev